mlserver/cli/main.py

```diff
@@ -47,13 +47,17 @@ async def start(folder: str):
 @root.command("build")
 @click.argument("folder", nargs=1)
 @click.option("-t", "--tag", type=str)
+@click.option("--cache/--no-cache", default=False)
```

Comment:
Shouldn't this default be `True` so that it aligns with the previous behaviour?

Reply:
Yeah I ummed and arred over this depends on how you want to use it and docker doesn't make it easy either, their usage is --no-cache since the cache is used by default. So if we want to mimic docker I guess we can make it into just --no-cache an then is_flag=True instead of the default=True but I think we would still need to handle since it's None or True surely? Let me test and adjust.